backend/clickhouse/migrations/000145_reacreate_errors_joined_vw.up.sql

```diff
@@ -0,0 +1,10 @@
+DROP VIEW IF EXISTS errors_joined_vw;
+CREATE VIEW IF NOT EXISTS errors_joined_vw AS
```

Comment:
Looks like we do not need to recreate row policies as they are on a different table, but wanted to double check